storage/index.go

```diff
@@ -350,14 +352,12 @@ func (i *nsIndex) writeBatchForBlockStartWithRLock(
 
 	// i.e. we have the block and the inserts, perform the writes.
 	result, err := block.WriteBatch(inserts)
+
```

Comment:
would be good to add the numDuplicates  to the result type returned from the blockWriteBatch call, and a metric for it.